datalad/crawler/pipelines/tests/test_balsa.py

```diff
@@ -124,6 +126,8 @@ def test_smoke_pipelines():
 @serve_path_via_http
 @with_tempfile
 @with_tempfile
+@skip_direct_mode  #FIXME
```

Comment:
note that if the entire module of tests failes (probably for issues not to be fixed in that module per se), might be worth just skip it at the module level, e.g. how it is done already for scrapy
```python
try:
    import scrapy
except ImportError:
    raise SkipTest("Needs scrapy")
```

Reply:
The entire point is to do it on a per-test level, so we are able to easily ensure we don't worsen the status quo and we can fix that stuff less complex steps. Have a look at #1562.

Reply:
Yes, please keep in mind that these are steps towards not having anything fail. We need a quick way to say if there is anything worth fixing at the level of and when touching individual tests. Please keep individual test labeling.